src/dbus_api/tree.rs

```diff
@@ -576,6 +605,38 @@ where
                 self.handle_pool_usage_change(item, new_usage);
                 Ok(true)
             }
+            DbusAction::FsBackgroundSizeChange(uuid, new_size) => {
```

Comment:
This isn't quite possible without reworking the macro. `background_arm! {}` is limited to use with property changes that are related to a UUID instead of a path. All other match arms (that's where the name comes from) take paths.